corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_blank.html

```diff
@@ -0,0 +1 @@
+<script type="text/html" id="blank-entry-ko-template"></script>
```

Comment:
I suppose for consistencies sake that that is ok 🤔

Reply:
Tell me more about what you don't like here?

(I don't like that it's a 3-line file, but not sure if you're talking about something else.)

Reply:
Yeah, that just seemed kind of silly. But it makes sense. I rather have that than a 700 line file

Reply:
agreed on both points